proof/invariant-abstract/AInvs.thy

```diff
@@ -29,13 +29,12 @@ lemma pred_tcb_at_upd_apply:
 
 lemma thread_set_tcb_arch_ct_schedulable[wp]:
   "thread_set (\<lambda>tcb. tcb\<lparr>tcb_arch := arch_tcb_context_set us (tcb_arch tcb)\<rparr>) t \<lbrace>ct_schedulable\<rbrace>"
+  apply (clarsimp simp: schedulable_def)
   apply (simp add: thread_set_def)
   apply (rule bind_wp[OF _ assert_get_tcb_ko'])
   apply (wpsimp wp: set_object_wp)
-  apply (fastforce simp: schedulable_def is_sc_active_def get_tcb_def ko_atD
-                         in_release_queue_def
-                  split: option.splits )
-  done
+  by (fastforce simp: obj_at_def opt_pred_def opt_map_def tcbs_of_kh_def scs_of_kh_def
+               split: option.splits)
```

Comment:
Those `cbs_of_kh_def scs_of_kh_def` are ones I meant with interfering with the simplifier.